src/utils/drag-and-drop-utils.js

```diff
@@ -68,6 +68,9 @@ function canDrop(dropTargetProps, monitor, isHover = false) {
 
     const targetDepth = getTargetDepth(dropTargetProps, monitor);
     const draggedNode = monitor.getItem().node;
+
+    const parentNode = dropTargetProps.rows[dropTargetProps.path[dropTargetProps.path.length - 2]];
```

Comment:
This won't work when the keys in the path are generated by a custom `getNodeKey`. It only works when the `treeIndex` is used as the node key.